### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by aliaksandr-s**
Please review this, @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: russian/sleep/brushing/bluebrush/blue-toothbrush.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 После того, как вы взяли в руку синюю щетку, комната обращается вверх тормашками и вы начинаете увязать в зефире.
-"Что вы будете делать?"
\ No newline at end of file
+"Что вы будете делать?"
+
+[Паниковать] (panich/panic.md)
```
Comment (casey-collab) [inline russian/sleep/brushing/bluebrush/blue-toothbrush.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
It looks like the files `russian/sleep/brushing/bluebrush/ask-for-help/ask-for-help.md` and `russian/sleep/brushing/bluebrush/panic/panic.md` are empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to each file, or delete the options that lead to them.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by aliaksandr-s** [comment]
@casey-collab, please review this again

**Comment by casey-collab** [comment]
Looks great! :sunglasses:

**Pull request merged**